Dockerfile

```diff
@@ -14,7 +15,7 @@ COPY . .
 ARG BUILD_TYPE='dev'
 
 RUN if [ "${BUILD_TYPE}" = "dev" ]; then \
-        pip install --no-cache-dir -e '.[checking, doctest, document, example, testing]'; \
+        pip install --no-cache-dir -e '.[checking, doctest, document, example, testing]' -f https://download.pytorch.org/whl/torch_stable.html; \
```

Comment:
[comment] `-f https://download.pytorch.org/whl/torch_stable.html` is in [the official installation document](https://pytorch.org/get-started/locally/).

```shell
pip install torch==1.5.0+cpu torchvision==0.6.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
```